backend/packages/Upgrade/src/loaders/typeormLoader.ts

```diff
@@ -1,23 +1,187 @@
+import { addExperimentType1665047953705 } from '../database/migrations/1665047953705-addExperimentType';
+import { factorialExperiment1671182276793 } from '../database/migrations/1671182276793-factorialExperiment';
+import { factorRestructing1679319498815 } from '../database/migrations/1679319498815-factorRestructing';
+import { replaceAliasWithPayload1679641063207 } from '../database/migrations/1679641063207-replaceAliasWithPayload';
+import { payloadError1684994998819 } from '../database/migrations/1684994998819-payloadError';
+import { conditionOrder1684996673747 } from '../database/migrations/1684996673747-conditionOrder';
+import { uniquifier1686575888877 } from '../database/migrations/1686575888877-uniquifier';
+import { remainingDevMigrations1692792001871 } from '../database/migrations/1692792001871-remainingDevMigrations';
+import { archivedState1692936809279 } from '../database/migrations/1692936809279-archivedState';
+import { stratificationFactorStatus1696829429134 } from '../database/migrations/1696829429134-stratificationFactorStatus';
+import { stratificationFactorFeature1696498128121 } from '../database/migrations/1696498128121-stratificationFactorFeature';
+import { addGroupIdForIndividualExclusion1710484793070 } from '../database/migrations/1710484793070-addGroupIdForIndividualExclusion';
+import { userDefaultRoleReader1713260614311 } from '../database/migrations/1713260614311-userDefaultRoleReader';
+import { Typeorm1719738784139 } from '../database/migrations/1719738784139-Typeorm';
+import { featureFlagSegmentInclusionExclusion1711566460836 } from '../database/migrations/1711566460836-featureFlagSegmentInclusionExclusion';
+import { featureFlagExposure1711569269846 } from '../database/migrations/1711569269846-featureFlagExposure';
+import { featureFlagContext1711569517358 } from '../database/migrations/1711569517358-featureFlagContext';
+import { featureFlagStatusTags1711652015345 } from '../database/migrations/1711652015345-featureFlagStatusTags';
+import { contextInMetric1712553037665 } from '../database/migrations/1712553037665-contextInMetric';
+import { featureFlagFilterMode1714680515570 } from '../database/migrations/1714680515570-featureFlagFilterMode';
+import { addingIndex1716191003726 } from '../database/migrations/1716191003726-addingIndex';
+import { monitoredIndexesForExport1718377498760 } from '../database/migrations/1718377498760-monitoredIndexesForExport';
+import { addTokenValidationFailedEnum1720515933833 } from '../database/migrations/1720515933833-addTokenValidationFailedEnum';
+import { featureFlagLists1720810654183 } from '../database/migrations/1720810654183-feature-flag-lists';
+import { removeGroupIdFromIndividualExclusion1721124249413 } from '../database/migrations/1721124249413-removeGroupIdFromIndividualExclusion';
+import { featureFlagFilterModeChangeDefault1722540825048 } from '../database/migrations/1722540825048-featureFlagFilterModeChangeDefault';
+
+const entities = [
```

Comment:
Could we not just use `env.app.dirs.entities` instead of having to define this and update it all the time? I tested it locally and it works.